README.md

```diff
@@ -1,16 +1,136 @@
-# Title
+# Programming as Conversation 2: Repetition With the `while` Statement
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+* Define the Basic `while...do...end` Structure
+* Write an Infinite Loop
+* Use Control-C to Break an Infinitely Looping Program
+* Terminate a `while...do...end` Loop Naturally
+* Terminate a `while...do...end` Loop With `break` Statement
 
 ## Introduction
 
-## SWBAT 1
+The final piece of using _statements_ to control the flow of Ruby execution is
+_repetition_. While the **default sequence** requires Ruby to execute top-down,
+left to right, we've seen that we can skip chunks of code using the _selection_
+statement `if...else...end`. In some ways the reverse of _selection_ is
+_repetition_: "Don't move on," we tell Ruby, "do something else until I say
+it's OK to move on." The most fundamental _repetition_ construct, present in
+pretty much every programming language, is the `while...do...end` loop.
 
-## SWBAT 2
+Ruby features other _repetition_ statements, but, for the most part, they're
+simplifications on the `while...do...end` statement.
+
+## Define the Basic `while...do...end` Structure
+
+The basic `while...do...end` looks like:
+
+```ruby
+while (condition expression) do
+ # stuff to do
+end
+```
+
+As long as the condition expression is `true` (or truthy), the code inside the
+`do...end` block will run. Keep in  mind, as an _expression_ the condition
+expression can be quite rich, using `&&`, `||`, `()`, etc.
+
+## Write an Infinite Loop
+
+Given this definition of `while...do...end`, any truthy expression will make
+the loop run forever.
+
+```ruby
+while true do
+  puts "say this forever..."
+end
+```
+
+Remember truthy? Here again:
+
+```ruby
+while -1 do # -1 is truthy....
+  puts "say this forever..."
+end
+```
+
+## Use Control-C to Break an Infinitely Looping Program
+
+If you ever accidentally have a program run inside a loop (maybe you made an
```

Comment:
this sentence isn't quite clear. Maybe something like:

If your running a program and find its stuck in an infinite loop